.github/workflows/fetchAssets.yaml

```diff
@@ -12,64 +12,61 @@ jobs:
       - name: Download assets
         run: |
           echo "REQUIRED_ASSETS=$REQUIRED_ASSETS"
-          IFS=':' read -r -a REQUIRED_ASSETS <<< "$REQUIRED_ASSETS"
-          RELEASE_URL='https://api.github.com/repos/NSoiffer/MathCATForPython/releases/latest'
-          RELEASE_DATA=$(curl -s -H 'Accept: application/vnd.github.v3+json' "$RELEASE_URL")
+          release_data=$(curl -s -H 'Accept: application/vnd.github.v3+json' "$RELEASE_URL")
 
-          if [ ! -d assets ] ; then
+          if [[ ! -d assets ]] ; then
             echo 'Directory "assets" does not exist. Creating it.'
             mkdir assets
           else
             echo 'Directory "assets" exists.'
           fi
 
-          for ASSET_NAME in "${REQUIRED_ASSETS[@]}"; do
-            echo "ASSET_NAME=$ASSET_NAME"
-            ASSET_URL=$(echo "$RELEASE_DATA" | jq -r ".assets[] | select(.name == \"$ASSET_NAME\") | .url")
-            if [[ "$ASSET_URL" != "null" ]]; then
-              echo "Downloading $ASSET_NAME"
-              curl -L -H "Accept: application/octet-stream" -o "$ASSET_NAME" "$ASSET_URL"
+          while IFS= read -r asset_name; do
+            if [[ "$asset_name" =~ ^[[:space:]]*$ ]]; then
+              continue
+            fi
+            echo "asset_name=$asset_name"
+            asset_url=$(echo "$release_data" | jq -r ".assets[] | select(.name == \"$asset_name\") | .url")
+            if [[ "$asset_url" != "null" ]]; then
+              echo "Downloading $asset_name"
+              curl -L -H "Accept: application/octet-stream" -o "$asset_name" "$asset_url"
             else
-              echo "Asset $ASSET_NAME not found in the latest release."
+              echo "Asset $asset_name not found in the latest release."
               exit 1
             fi
-          done
-      - name: Delete previously downloaded assets
-        run: |
-          shopt -s nullglob
-          for file in assets/*; do
-            echo "Removing $file"
-            rm -rf "$file"
-          done
-      - name: Unzip assets
-        run: |
-          IFS=':' read -r -a REQUIRED_ASSETS <<< "$REQUIRED_ASSETS"
-          for ASSET_NAME in "${REQUIRED_ASSETS[@]}"; do
-            unzip -o "$ASSET_NAME" -d assets
-            rm -f "$ASSET_NAME"
-          done
+            unzip -o "$asset_name" -d assets
+            rm -f "$asset_name"
+          done <<< "$REQUIRED_ASSETS"
       - name: Commit and push downloaded assets
         run: |
-          echo 'About to diff, commit, and push ...'
-          ls -r
-          ls -r assets
-          # git status
           git add assets
           git diff --cached --quiet && echo "No changes to commit" && exit 0
           git config --global user.name "github-actions"
           git config --global user.email "github-actions@github.com"
           git commit -m "Add downloaded assets"
           git push
+      - name: Create issue in nvda repo using gh
+        env:
+          GH_TOKEN: ${{ secrets.NVDA_ISSUE_TOKEN }}
```

Comment:
ah I didn't realise a token was needed for creating issues. This is surprising. Given a token is needed anyway, could you change this to creating a pull request updating the commit used? In the PR body, please add a reminder to add a changelog entry when merging.